go.mod

```diff
@@ -3,20 +3,21 @@ module github.com/open-cluster-management/governance-policy-status-sync
 go 1.16
 
 require (
-	github.com/onsi/ginkgo v1.14.1
-	github.com/onsi/gomega v1.10.1
-	github.com/open-cluster-management/addon-framework v0.0.0-20210415135036-d591b71789d6
-	github.com/open-cluster-management/governance-policy-propagator v0.0.0-20210330231547-932d5d73e0c3
-	github.com/operator-framework/operator-sdk v0.19.4
+	github.com/onsi/ginkgo v1.16.4
+	github.com/onsi/gomega v1.13.0
+	github.com/open-cluster-management/addon-framework v0.0.0-20210621074027-a81f712c10c2
+	github.com/open-cluster-management/governance-policy-propagator v0.0.0-20211012174109-95c3b77cce09
 	github.com/spf13/pflag v1.0.5
-	k8s.io/api v0.20.5
-	k8s.io/apimachinery v0.20.5
+	k8s.io/api v0.22.1
+	k8s.io/apimachinery v0.22.1
 	k8s.io/client-go v12.0.0+incompatible
 	k8s.io/klog v1.0.0
-	sigs.k8s.io/controller-runtime v0.6.2
+	open-cluster-management.io/addon-framework v0.0.0-20211101093604-8c0b8f52ad78
+	sigs.k8s.io/controller-runtime v0.9.2
 )
 
 replace (
 	github.com/go-logr/zapr => github.com/go-logr/zapr v0.4.0
-	k8s.io/client-go => k8s.io/client-go v0.20.5
+	github.com/open-cluster-management/multicloud-operators-placementrule => github.com/open-cluster-management/multicloud-operators-placementrule v1.2.4-0-20210816-699e5.0.20211012154812-5fac6c25d2f6
```

Comment:
Is this needed for upstream?

Reply:
It pins a specific version of a nested dependency... so I would guess it is still required. I'll test locally.

Reply:
Building without it gives a failure.